docker/mysql/Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM mysql:5.7
+FROM mysql:5.7.27
 ENV MYSQL_ROOT_PASSWORD=location
-ENV MYSQL_DATABASE=location
```

Comment:
This line caused the container to create the `location` database at startup. We now have code that creates/drops databases, so we don't need this anymore.